File: gherkin/java/src/main/java/gherkin/pickles/PickleLocation.java
package gherkin.pickles;

public class PickleLocation {
    private final int line;
    private final int column;

    public PickleLocation(int line, int column) {
        this.line = line;
        this.column = column;
    }

    public int getLine() {
        return line;
    }

    public int getColumn() {
        return column;
    }
}

File: gherkin/dotnet/Gherkin/Pickles/PickleLocation.cs
namespace Gherkin.Pickles
{
    public class PickleLocation
    {
        public int Line { get; private set; }
        public int Column { get; private set; }

        public PickleLocation(int line, int column)
        {
            Line = line;
            Column = column;
        }
    }
}
